### DISCUSSION TITLE
Frequently Asked Questions

### DISCUSSION THREAD

**Discussion by aprilnovak** [neams-th-coe/cardinal#271] [Announcements]
## I am having issues building/installing the code
Please see our [Getting Started](https://cardinal.cels.anl.gov/start.html) page. If you are having issues specifically with MOOSE, please consult the MOOSE [Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) page. You do *not* need to individually build MOOSE, OpenMC, or NekRS when using Cardinal; but if you are encountering errors, you may also find useful information in the code-specific installation instructions:

- [MOOSE build instructions](https://mooseframework.inl.gov/getting_started/installation/index.html)
- [OpenMC build instructions](https://docs.openmc.org/en/develop/quickinstall.html)
- [NekRS build instructions](https://github.com/Nek5000/nekRS)

For general troubleshooting and common errors, please see our [Help](https://cardinal.cels.anl.gov/faq.html) page. If you still encounter difficulties, please post a question in the Discussions. 

## Where do I find Cardinal documentation?
Please see our [Documentation](https://cardinal.cels.anl.gov/source/index.html) for descriptions of how to use Cardinal, and our [Tutorials](https://cardinal.cels.anl.gov/tutorials/index.html) for detailed examples. 

## Where do I find OpenMC documentation?
Please see the [OpenMC documentation](https://docs.openmc.org/en/develop/).

## Where do I find NekRS documentation?
Please see the [NekRS documentation](https://nekrsdoc.readthedocs.io/en/latest/index.html).

## How do I contribute?
Cardinal uses a [fork-and-pull](https://reflectoring.io/github-fork-and-pull/) workflow, where you commit work on a fork of Cardinal, and then merge into the main repository when ready for review. Please see these [instructions](https://cardinal.cels.anl.gov/contributing.html) for how to contribute to Cardinal. If you try to push directly to the main repository, you will see an error like

```
ERROR: Permission to neams-th-coe/cardinal.git denied to username.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

This means that you need to push to your fork, not the main repository.

**Comment by wangshuaiwu** [comment]
![7cdf1e396a6560628d6429325382495](https://user-images.githubusercontent.com/122420896/234238163-b9b6bb5d-ef97-4754-b59b-bb107d7a5330.jpg)
When I logged in to my user today, I found that all the application ICONS under my user turned into this look. When I logged in to other users, their application ICONS were normal. When I created a new user, his application ICONS were also normal. So I created a new user and re-installed the cardinal on moose. When I finished the installation and logged out again, the icon disappeared again. I think the reason may be the installation of cardinal,I also installed cardinal without moose in other users, and its app ICONS are fine，Can you help me? Do you know what this is about @aprilnovak 
My OS is Rocky9.1,This is the content of my.Bashrc
![image](https://user-images.githubusercontent.com/122420896/234247641-34289de9-82d3-4424-900e-c3f58e69540c.png)

**Reply by aprilnovak** [reply]
Hi @wangshuaiwu I am not sure on this - is this WSL? This is not really the territory of Cardinal, I think this is something more systems-related for you. 

In the future, please post questions as new topics in the Discussions - this post is meant just for FAQs, not specific user assistance.
